PixyViSy.cpp

```diff
@@ -15,48 +15,77 @@ void PixyViSy::update(void)
   uint8_t all_blocks_count = pixy.getBlocks();
   left_pixels = 0;
   right_pixels = 0;
-  action = 'N';
+  action = 'N'; // as "Nothing"
   distance = ~0;
   uint16_t goal_pix_height = 0;
+  uint8_t max[3] = { all_blocks_count, all_blocks_count, all_blocks_count };
```

Comment:
@vargac if `all_blocks_count` is the number of blocks, will `pixy.blocks[max[0]]` for instance be defined? I do not think so.